### PR TITLE
[COST-5390] Make downstream v3.3.1

### PR DESCRIPTION
**Pull request by djnakabaale**

### CODE REVIEW

File: costmanagement-metrics-operator/3.3.1-test-1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM scratch
```
Comment (maskarb) [inline costmanagement-metrics-operator/3.3.1-test-1/Dockerfile:1]: This directory is wrong. It should just be `3.3.1`, not `3.3.1-test-1`

Comment (djnakabaale) [reply costmanagement-metrics-operator/3.3.1-test-1/Dockerfile]: Oof, I should use a diff terminal tab for testing things. 
Thanks for spotting this one!

File: costmanagement-metrics-operator/3.3.1/manifests/costmanagement-metrics-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -608,8 +612,5 @@ spec:
   minKubeVersion: 1.24.0
   provider:
     name: Red Hat
-  version: 3.3.0
-  relatedImages:
-    - name: koku-metrics-operator
-      image: quay.io/project-koku/koku-metrics-operator@sha256:0c8091f0c04593c19249755bde418cc09e5d30cc55c6cbff2c5b60bdf60dfde9
```
Comment (maskarb) [inline costmanagement-metrics-operator/3.3.1/manifests/costmanagement-metrics-operator.clusterserviceversion.yaml:614]: Just FYI, in regards to Konflux, this `relatedImages` is the thing that the docs mention. Brew automatically adds this to the CSV, but Konflux does not.

**Review by maskarb — APPROVED** [review]

File: costmanagement-metrics-operator/3.3.1/manifests/costmanagement-metrics-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (djnakabaale) [reply costmanagement-metrics-operator/3.3.1/manifests/costmanagement-metrics-operator.clusterserviceversion.yaml:614]: 👍🏾 okay, sounds good!

**Pull request merged**